facebook.js

```diff
@@ -1,78 +1,148 @@
+var stad = chrome.i18n.getMessage("stand");
+var per = chrome.i18n.getMessage("per");
```

Comment:
I agree with the other comment that says chrome.i18n is probably the wrong way to do this. The user's chrome locale and the facebook locale aren't necessarily the same. You should look up the translation based on the facebook locale, not the current chrome locale.